include/deal.II/lac/la_parallel_vector.h

```diff
@@ -445,6 +445,15 @@ namespace LinearAlgebra
       void
       swap(Vector<Number, MemorySpace> &v);
 
+      /**
+       * Move assignment operator. Uses the swap method.
+       *
+       * @note Since the swap() method can throw exceptions (should some MPI
+       * checks fail) this function is not noexcept.
```

Comment:
I don't like this comment because it talks about implementation details. *How* the function is implemented should not matter to the reader. *What it does* is the important point. So I would phrase this as follows:
```suggestion
       * @note This method may throw exceptions (should some MPI
       * checks fail) and is consequently not `noexcept`.
```

Reply:
I just saw this review! Yes, that's the correct order. Fixed.